samples/matter/manufacturer_specific/README.rst

```diff
@@ -0,0 +1,275 @@
+.. _matter_manufacturer_specific_sample:
+
+Matter: Manufacturer specific
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates an implementation of custom Manufacturer specific clusters used by application layer.
+This sample utilizes Development Kit's buttons and LEDs to demonstrate the functionality of the custom ``NordicDevkit`` cluster.
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
```

Comment:
```suggestion
The device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
```